### DISCUSSION TITLE
저커버그도 따라야 하는 메타(META)의 피드백 원칙 - EO 채널 영상

### DISCUSSION THREAD

**Discussion by TaeHyoungKwon** [ThinkAboutSoftware/AcademicConference#289] [General]
https://www.youtube.com/watch?v=Fn3fGp9s3bY

실리콘밸리의 팀장들 책 보면서 피드백 관련된 내용이 많이 나왔는데, 이 내용도 한번 보면 좋을것 같아서 공유해봅니다~

**Comment by TaeHyoungKwon** [comment]
![스크린샷 2023-03-27 오후 10 54 30](https://user-images.githubusercontent.com/18538418/227960260-37e223e1-7832-4411-9dd2-81fb209fc565.png)

영상 보고 댓글보는데... 이런 매운 댓글이 있네요 / 참고정도 하면 좋을것 같습니다~

**Comment by fkdl0048** [comment]
어우 책에 연관된 내용까지 정말 감사합니다..! (솔직한 칭찬입니다..^^)
